### PR TITLE
Apply operators conditionally based on commands

### PR DESCRIPTION
**Pull request by metonymic-smokey**
Signed-off-by: metonymic-smokey <ahuja.aditi@gmail.com>

**Description**
This applies all three manifests on system start, but only the operator manifest on system stop.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by metonymic-smokey** [comment]
@navendu-pottekkat 
Here are the screenshots for platform kubernetes(the endpoint issue is invalid):
1. start:
image

2. status:
image

3. stop:
image

**Comment by metonymic-smokey** [comment]
@navendu-pottekkat here are the screenshots for docker:
1. start
image

2. pod status
image
 
3. stop
image

File: mesheryctl/pkg/utils/platform.go [hunk 1]
```diff
@@ -311,6 +324,29 @@ func ApplyOperatorManifest(client *meshkitkube.Client, update bool, delete bool)
 		return err
 	}
 
+	//condition to check for system stop
+	if delete == false {
```
Comment (navendu-pottekkat) [inline mesheryctl/pkg/utils/platform.go:328]:
```suggestion
	if !delete {
```

**Review by navendu-pottekkat — APPROVED** [review]
Looks Good! Thank you @metonymic-smokey 🚀

File: mesheryctl/pkg/utils/platform.go (hunk 1 above)
Comment (navendu-pottekkat) [reply mesheryctl/pkg/utils/platform.go:328]: Will make this change in an upcoming PR

**Pull request merged**